src/includes/class-wordlift-configuration-service.php

```diff
@@ -430,4 +430,17 @@ public function get_batch_analysis_url() {
 
 	}
 
+	/**
+	 * Get the URL to perform autocomplete request.
+	 *
+	 * @since 3.15.0
+	 *
+	 * @return string The URL to call to perform the batch analyzes.
+	 */
+	public function get_autocomplete_url() {
+
+		return WL_CONFIG_WORDLIFT_API_URL_DEFAULT_VALUE . '/autocomplete';
```

Comment:
I am not sure that all external API's will have the same structure, so this may need to be changed